### PR TITLE
Sanity Check Install Content

### LINKED ISSUE #3537: Sanity-check Install-related content
## Description: 
We want to make sure the install-related content which lives in the Alpha site (https://github.com/knative/docs/tree/mkdocs) meets a sanity-check. 

In the new site, these install instructions will be for administrators  _specifically_, which is why the story is written from an admin's perspective. (See: #3525 and PR #3526)

---

## Story
```
As an Amanda the Administrator 
I want to go through the pages under the "Installing Knative" section of the site 
And feel like some someone has done a cursory review of this section 
So that I assign more credibility to the Knative site 
```

## Acceptance Criteria
`Navigation`
```
GIVEN I find my way to https://julz.github.io/docs/dev/install/ 
WHEN I look at the page 
THEN I see the Pre-Requisites Section front and center
AND I see there is a clear navigation to the left of the content
 AND none of the page titles are potentially confusing 
```

`Legibility, Credibility Check`
```
GIVEN I find my way to any of the pages in the sub-directory https://julz.github.io/docs/dev/install/ 
WHEN I read through the pages 
THEN I do not find any broken links 
AND I do not find any typos
AND the way the content is organized on the page makes sense
AND there is nothing wonky looking in this section of the site 
```

## Known Issues and Considerations: 
- [x] When I access {url}/docs/dev/install it takes me to an install page without a navigation
- [x] The table of contents currently has a section called "Installation Files" which is confusing since really it is a description of the installation files 
- [x] Anywhere we are "Verifying the Output" should use the custom CSS { .bash .no-copy } as seen under "Expected Output" here: https://julz.github.io/docs/dev/getting-started/first-service/#run-your-knative-service

### PR DESCRIPTION
**Pull request by omerbensaadon**
Fixes #3537

### CODE REVIEW

File: docs/install/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ showlandingtoc: "false"
 # Installing Knative
 
 !!! tip
-    If you're looking for an easy way to install a local distribution of Knative for prototyping, check out our <a href="https://github.com/csantanapr/knative-kind" target="_blank">Quick Install w/ KonK</a>
+    If you're looking for an easy way to install a local distribution of Knative for prototyping, check out our [Getting Started guide](../getting-started/getting-started.md){_blank}
```
Comment (abrennan89) [inline docs/install/README.md:27]:
```suggestion
    You can install a local distribution of Knative for development use, by following the [Getting started guide](../getting-started/getting-started.md){_blank}.
```

File: docs/install/install-eventing-with-yaml.md [hunk 2]
```diff
@@ -131,7 +132,7 @@ Follow the procedure for the broker of your choice:
         kubectl apply -f {{ artifact(org="knative-sandbox",repo="eventing-kafka-broker",file="eventing-kafka-broker.yaml")}}
         ```
 
-    For more information, see the [Kafka broker](./../eventing/broker/kafka-broker.md) documentation.
+    For more information, see the [Kafka broker](./../eventing/broker/kafka-broker/README.md) documentation.
```
Comment (abrennan89) [inline docs/install/install-eventing-with-yaml.md:135]:
```suggestion
    For more information, see the [Kafka broker](./../eventing/broker/kafka-broker/) documentation.
```
Pretty sure you can just leave off the default landing page .md file in these kinds of links but idk if this works the same for mkdocs?

File: mkdocs.yml [hunk 3]
```diff
@@ -22,22 +22,35 @@ nav:
           - Creating your first Trigger: getting-started/first-trigger.md
         - What's Next?: getting-started/next-steps.md
     - Installing Knative:
```
Comment (abrennan89) [inline mkdocs.yml:24]: Maybe move this straight into an admin guide?

**Comment by omerbensaadon** [comment]
This is a really solid question… I… think so…? Not at my PC rn but if it is
possible I’ll remove READMEs before merging :)

**Review by abrennan89 — COMMENTED** [review]
Some comments / suggestions

File: mkdocs.yml (hunk 3 above)
Comment (omerbensaadon) [reply mkdocs.yml:24]: The Eventing Getting Started? I view this as a developer persona thing

Comment (abrennan89) [reply mkdocs.yml:24]: Nah the install docs :)

Comment (omerbensaadon) [reply mkdocs.yml:24]: yes that is 100% admin guide material

**Comment by omerbensaadon** [comment]
@abrennan89 for some reason removing the READMEs in the "Install optional extensions" page causes 404s... huh.

**Comment by omerbensaadon** [comment]
@abrennan89 oops can you lgtm

**Review by abrennan89 — APPROVED** [review]
/lgtm

Can you open a bug in docs or UX for the README.md thing breaking some pages but not others? Would be good to fix any issues hindering consistency.

**Comment by omerbensaadon** [comment]
@abrennan89 I'll throw it on my list!

**Pull request merged**